app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb

```diff
@@ -85,9 +85,10 @@ def uuid_provider_types
       def fetch_server_entities
         @data[:middleware_deployments] = []
         @data[:middleware_datasources] = []
+        @data[:middleware_jms] = []
         @eaps.map do |eap|
-          @ems.child_resources(eap.path).map do |child|
-            next unless child.type_path.end_with?('Deployment', 'Datasource')
+          @ems.child_resources(eap.path, true).map do |child|
+            next unless child.type_path.end_with?('Deployment', 'Datasource', 'JMS%20Topic', 'JMS%20Queue')
```

Comment:
In hawkular the path contains spaces, and they are encoded as %20 (JMS Topic -> JMS%20Topic), I thought about using `child.type_path.includes("JMS") && (child.type_path.includes("Topic") || child.type_path.includes("Queuq"))` but the rule with `ned_with` felt stronger and consistent. But fi you think it would be better to use contains, I can rewrite it, no problem.